dynawo/sources/Models/Modelica/Dynawo/Examples/ENTSOE/TestCase1.mo

```diff
@@ -118,7 +126,7 @@ equation
     Line(points = {{38, 10}, {70, 10}, {70, 6}, {78, 6}}, color = {0, 0, 127}));
   connect(generatorSynchronous.omegaPu_out, pss.omegaPu) annotation(
     Line(points = {{38, -6}, {78, -6}}, color = {0, 0, 127}));
-  connect(pss.UpssPu, avr.UpssPu) annotation(
+  connect(pss.VPssPu, avr.UpssPu) annotation(
```

Comment:
I didn't find VPssPu variable.

We can keep it UpssPu

Reply:
Indeed, this belongs to the branch 3091_use_new_standard_regulations